docs/lua/classes/ClientConnection.md

```diff
@@ -0,0 +1,2 @@
+# Class - ClientConnection
+* <pre>ClientConnection.user:[User](User.md)</pre> - User Associated with this client connection
```

Comment:
Markdown links don't work in `<pre>` or will render inconsistently in different places

Reply:
Fixed